feature-libs/cart/quick-order/styles/_quick-order-table.scss

```diff
@@ -143,5 +143,13 @@ cx-quick-order-table {
 
   .cx-quick-order-table-item {
     margin: 0;
+
+    .cx-quick-order-table-item-product-image {
+      .img-missing {
+        display: block;
+        width: 65px;
```

Comment:
May you please change the value as I have checked and <a> tag with image has always 63,5px width and height